app/javascript/components/api/api.js

```diff
@@ -25,8 +25,12 @@ const createAPI = () => {
     return response;
```

Comment:
Responses have common structure `{data: DATA}`
maybe it will be clearly to `return response['data']` here instead of doing it in each request?

as you've already done in `onFail()`